spec/models/room_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 RSpec.describe Room, type: :model do
   it '#Room valid with all attributes' do
-    room = create(:room, number: 101, room_type: 0, status: 0)
+    room = create(:room, number: 101, room_type: 0, room_status: 0)
```

Comment:
Why not extending a bit the factory to add a `available_room` and `occupied_room` factories?

Reply:
Also fun fact, since you set the status as `enum`, you can also create the `status` with `:available` instead of `0`. It saves you to having to look up what index position each status is.